translator/trace/zipkin/zipkinv1_to_protospan.go

```diff
@@ -266,19 +289,27 @@ func parseZipkinV1Annotations(annotations []*annotation) *annotationParseResult
 		case "cs":
 			fallthrough
 		case "cr":
-			if res.Kind == tracepb.Span_SPAN_KIND_UNSPECIFIED {
-				res.Kind = tracepb.Span_CLIENT
-			}
-			fallthrough
+			res.Kind = tracepb.Span_CLIENT
```

Comment:
@owais can I have your eye on these changes, please? It used fallbacks liberaly and I got rid of most. This changes the behavior slightly, and particularly how the Endpoint is set outside the switch now.
I think the new coder is correct, but want to know your opinion if the old code handled some edge cases which I should care about and which now work differently because of my changes.

Reply:
I don't know if in practice this will matter much but since there can be multiple annotations on a single span this is a change in behavior: the kind before was derived from the first one. You can avoid the `fallthrough` by having `case "cs", "cr":`

Reply:
Fixed.